services/dumper.js

```diff
@@ -200,7 +201,10 @@ function Dumper(config) {
     copyHandleBarsTemplate({
       source: 'app/forest/collection.hbs',
       target: `forest/${tableToFilename(table)}.js`,
-      context: { isMongoDB: config.dbDialect === 'mongodb', table },
+      context: {
+        isMongoDB: config.dbDialect === 'mongodb',
+        table: stringUtils.pascalCase(stringUtils.transformToSafeString(table), false),
```

Comment:
As this pattern is present several time, you should definitely now consider a "business oriented" formatter that uses technical formatters:
```javascript
function getModelNameFromTableName(table) {
  return stringUtils.pascalCase(stringUtils.transformToSafeString(table), false);
}
```
It will definitely ease the future maintenance of this logic.